packages/graphback/src/resolvers/knex/resolverImplementation.ts

```diff
@@ -22,13 +23,14 @@ export const findAllTemplate = (fieldName: string, tableName: string): string =>
 
 export const findTemplate = (fieldName: string, tableName: string): string => {
     return `${fieldName}: (root, args, context, info) => {
-      return ${knexContext}.select().from('${tableName}').where(...${argumentContext}.fields)
+      return ${knexContext}.select().from('${tableName}').where(${argumentContext}.fields)
     }`
 }
 
 export const updateTemplate = (fieldName: string, tableName: string): string => {
     return `${fieldName}: (root, args, context, info) => {
-      return ${knexContext}('${tableName}').where('id', '=' , ${argumentContext}.id).update(${argumentContext}.input).then( () => {
-        return ${knexContext}.select().from('${tableName}').where('id', '=' , ${argumentContext}.id);
+      return ${knexContext}('${tableName}').where('id', '=' , ${argumentContext}.id).update(${argumentContext}.input).then( async () => {
```

Comment:
${argumentContext} is being used here but we actually have `args` above. This is looks really strange. We can hardcode args as we define it.